src/jsonapi-datastore.js

```diff
@@ -100,6 +101,17 @@ class JsonApiDataStore {
    * @param {object} model The model to destroy.
    */
   destroy(model) {
+    this.graph[model._type][model.id]._associations.map(function(assoc) {
+      if (self.graph[assoc.type][assoc.id][assoc.relation].constructor === Array) {
+        self.graph[assoc.type][assoc.id][assoc.relation].forEach(function(val, idx) {
+          if (val.id === model.id) {
+            self.graph[assoc.type][assoc.id][assoc.relation].splice(idx, 1);
+          }
+        });
+      } else if (self.graph[assoc.type][assoc.id][assoc.relation].id === model.id) {
+        self.graph[assoc.type][assoc.id][assoc.relation] = new JsonApiDataStoreModel(self.graph[assoc.type][assoc.id][assoc.relation]._type);
```

Comment:
It certainly could be null. I think I originally had it as null, but due to how we are using the data store, we found it useful to have the blank model there. I guess it could go either way. For a more global approach do you think null would be best?